src/main/java/seedu/duke/exceptions/IncorrectDeadlineFormatException.java

```diff
@@ -0,0 +1,7 @@
+package seedu.duke.exceptions;
+
+/**
+ * Signals that the user has input the date/time wrongly.
+ */
+public class IncorrectDeadlineFormatException extends Exception {
```

Comment:
Maybe later we should create a ZeroNoteException class that all the regular exceptions will inherit from? For now this should be fine though.

Reply:
added as a `//TODO`